DFC.App.Banners.Data/Models/CmsApiModels/PageBannerContentItemApiDataModel.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using System.Diagnostics.CodeAnalysis;
+
 using DFC.Content.Pkg.Netcore.Data.Models;
 
+using Newtonsoft.Json;
+
 namespace DFC.App.Banners.Data.Models.CmsApiModels
 {
     [ExcludeFromCodeCoverage]
     public class PageBannerContentItemApiDataModel : BaseContentItemModel
     {
-        // TODO: add properties.
+        [JsonProperty("PageUrl")]
```

Comment:
Please add the following properties
        [JsonProperty(PropertyName = "banner_WebPageURL")]
        public string? PageLocation { get; set; }

        [JsonProperty(PropertyName = "banner_WebPageName")]
        public string? WebPageName { get; set; }

Reply:
That's an outdated file.